src/support/json.h

```diff
@@ -403,6 +409,74 @@ struct Value {
     assert(isObject());
     return obj->count(x) > 0;
   }
+
+private:
+  // If the string has no escaped characters, setString() the char* directly. If
+  // it does require escaping, do that and intern a new string with those
+  // contents.
+  void unescapeAndSetString(char* str) {
```

Comment:
Since we do the json escaping in string.cpp, maybe we can do the unescaping there, too? That's where all the necessary unicode utilities are, too.

Reply:
Good idea, done.